modules/graphql_core/src/Plugin/GraphQL/Fields/EntityQuery/EntityQuery.php

```diff
@@ -297,6 +301,13 @@ protected function buildFilterConditions(QueryInterface $query, array $filter) {
     // Apply filter conditions.
     $conditions = !empty($filter['conditions']) ? $filter['conditions'] : [];
     foreach ($conditions as $condition) {
+      // Set the condition to enabled by default.
+      $enabled = (isset($condition['enabled'])) ? $condition['enabled'] : true;
```

Comment:
We support "default" values in the annotation. I think it would be better to use that:

```
"enabled" =  {
   "default"  = TRUE
   "type" = "Boolean"
}
```